dev/gen_protos.sh

```diff
@@ -5,7 +5,7 @@ if ! cargo install --list | grep "protoc-gen-prost-crate" > /dev/null; then
         exit 1
     fi
 fi
-if ! buf generate https://github.com/xmtp/proto.git#branch=nmolnar/mls-verification-service,subdir=proto; then
```

Comment:
this is temp, will make a companion PR in proto once this is finished

Reply:
I also have a TODO item to get all of our MLS protos into `main`, which'll help clean this up